core/src/main/java/io/grpc/internal/ServerCallImpl.java

```diff
@@ -191,6 +234,17 @@ public String getAuthority() {
   }
 
   /**
+   * Close the {@link ServerStream} because an internal error occurred. Allow the application to
+   * run until completion, but silently ignore interactions with the {@link ServerStream} from now
+   * on.
+   */
+  private void internalClose(Status internalError) {
+    internalClosed = true;
+    log.log(Level.FINE, internalError.getDescription(), internalError.asException());
```

Comment:
I don't think we need to really be logging this. But even if we are, it should be protected with an if isloggable check to avoid creating the exception.